scalding-serialization-macros/src/test/scala/com/twitter/scalding/serialization/macros/MacroOrderingProperties.scala

```diff
@@ -572,6 +602,14 @@ class MacroOrderingProperties extends FunSuite with PropertyChecks with ShouldMa
     checkCollisions[TestCC]
   }
 
+  test("Test out Sealed Trait") {
```

Comment:
don't we want one that does:

```scala
  check[SealedTraitTest]
```
the test below looks like it is just using the case class stuff you had before.